src/In.ProjectEKA.OtpService/Notification/NotificationService.cs

```diff
@@ -15,14 +15,23 @@ public NotificationService(ISmsClient smsClient)
         }
 
         public async Task<Response> SendNotification(Notification notification)
-        {
-            return notification.Action switch
+        { 
+            switch(notification.Action)
             {
-                Action.ConsentRequestCreated => await smsClient.Send(
-                    notification.Communication.Value,
-                    GenerateConsentRequestMessage(notification.Content)),
-                _ => new Response(ResponseType.InternalServerError, "")
-            };
+                case Action.ConsentRequestCreated:
+                    await smsClient.Send(
+                        notification.Communication.Value,
+                        GenerateConsentRequestMessage(notification.Content));
+                    return new Response(ResponseType.InternalServerError, "");
+
+                case Action.CmidRecovered:
+                    await smsClient.Send(
+                        notification.Communication.Value,
```

Comment:
You are supposed to call another method.

Reply:
Corrected